concrete/blocks/core_container/controller.php

```diff
@@ -16,50 +15,90 @@
 
 class Controller extends BlockController
 {
+    /**
+     * @var int|null
+     */
+    public $containerInstanceID;
+
+    /**
+     * @var string
+     */
     protected $btTable = 'btCoreContainer';
+
+    /**
+     * @var bool
+     */
     protected $btIsInternal = true;
+
+    /**
+     * @var bool
+     */
     protected $btIgnorePageThemeGridFrameworkContainer = true;
-    
-    public $containerInstanceID;
-    
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return string
+     */
     public function getBlockTypeDescription()
     {
-        return t("Proxy block for theme containers added through the UI.");
+        return t('Proxy block for theme containers added through the UI.');
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @return string
+     */
     public function getBlockTypeName()
     {
-        return t("Container");
+        return t('Container');
     }
-    
-    public function getContainerInstanceObject() :? Container\Instance
+
+    /**
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
+     *
+     * @return Container\Instance|null
+     */
+    public function getContainerInstanceObject(): ?Container\Instance
     {
         $entityManager = $this->app->make(EntityManager::class);
         if ($this->containerInstanceID) {
-            $instance = $entityManager->find(Container\Instance::class, $this->containerInstanceID);
-            return $instance;
+            return $entityManager->find(Container\Instance::class, $this->containerInstanceID);
         }
+
         return null;
     }
-    
+
+    /**
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
+     *
+     * @return void
+     */
     public function view()
     {
-        $template = null;
         $instance = $this->getContainerInstanceObject();
         if ($instance) {
             $container = $instance->getContainer();
-            if ($container) {
```

Comment:
Any reason why you removed this?

Reply:
Because it would always be true, https://github.com/deek87/concrete5/blob/317572a40edb252e017484542210cd48a8cf62be/concrete/src/Entity/Page/Container/Instance.php#L23-L27

getContainer will always return a container instance